common/src/main/java/io/github/gaming32/bingo/network/BingoNetwork.java

```diff
@@ -6,7 +6,7 @@
 import net.minecraft.resources.ResourceLocation;
 
 public class BingoNetwork {
-    public static final int PROTOCOL_VERSION = 7;
+    public static final int PROTOCOL_VERSION = 8;
```

Comment:
I'm not sure what it's sensible to do with protocol versions when updating to a new version, could even reset it back to 0. But it doesn't matter.

Reply:
I bumped it because the mod's network did actually change. Specifically, `Component` is now serialized to NBT.